roles/openshift_logging/tasks/main.yaml

```diff
@@ -28,6 +28,8 @@
   register: local_tmp
   changed_when: False
   check_mode: no
+  delegate_to: localhost
+  become: no
```

Comment:
@ewolinetz @mtnbikenc Do we have a general strategy for use of local_action versus delegate_to? Feels like we should be using one or the other rather than mixing them.